templates/generic/digest.html

```diff
@@ -0,0 +1,89 @@
+{{period}} github digest ({{#topic}}{{.}}{{/topic}}{{^topic}}{{#repos}}{{shortname}}{{^last}}, {{/last}}{{/repos}}{{/topic}})
```

Comment:
I think I'll get the subject from the plain text template, so need to keep it here.